php/spec/module-phar_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'spec_helper.rb'
+
+describe 'php::module-phar' do
+
+  let(:runner)    { ChefSpec::Runner.new }
+  let(:chef_run)  { runner.converge(described_recipe) }
+  let(:node)      { runner.node }
+
+  it 'adds ppa mirror configuration' do
+    expect(chef_run).to include_recipe('apt::easybib')
```

Comment:
@till Yes, Bacce came up with this question yesterday, too. Additionally, `apt::ppa` is a part of `apt::easybib`, too, so it is somewhat redundant. We opted to keep it this way for now so that the tests are all in the same style and refactoring and also the new test setup are in a consistent shape, and wrap this up afterwards when we also know that we do not have to add/change something while migrating the last cookbooks. :)